client/me/security-2fa-disable/index.jsx

```diff
@@ -151,18 +151,18 @@ class Security2faDisable extends Component {
 					</p>
 					<p>
 						{ translate(
-							'This will also disable your Application Passwords, ' +
+							'This will also disable your application passwords, ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'This will also disable your Application Passwords, though you can access them again if you ever re-enable Two-Step Authentication. If you decide to re-enable Two-Step Authentication, keep in mind you'll need to generate new backup codes.' )` *ES Score: 13*